pkgs/by-name/si/signal-desktop-bin/generic.nix

```diff
@@ -246,6 +246,8 @@ stdenv.mkDerivation rec {
       --replace-fail "/${bindir}/signal-desktop" ${meta.mainProgram} \
       --replace-fail "StartupWMClass=Signal" "StartupWMClass=signal"
 
+    mv $out/share/applications/signal{-desktop,}.desktop
```

Comment:
I am not aware of this notation yet !

Reply:
It's a bash thing and can be used with anything and is resolve before the argument is passed to bash.